.vscode/settings.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
add .vscode to .gitignore

Reply:
yup, will include after I get the basic stuff working